package.json

```diff
@@ -30,10 +30,16 @@
     "docs": "pushstate-server docs"
   },
   "lint-staged": {
-    "src/**/*.*.js": [
+    "src/**/*.js": [
       "prettier --no-semi --single-quote --trailing-comma all --write",
       "eslint src --fix",
       "build:docs",
+      "git add docs",
+      "git add"
+    ],
+    "src/**/*.*.js": [
+      "prettier --no-semi --single-quote --trailing-comma all --write",
+      "eslint src --fix",
```

Comment:
I think passing path isn't necessary here.. because lint-staged runs against staged files and src dir is already passed as filter
<br />
![screen shot 2017-05-25 at 23 02 59](https://cloud.githubusercontent.com/assets/22868432/26458796/89298054-419e-11e7-8199-d78ea8a1799b.png)
*from **lint-staged** docs*